utils/p4runtime_lib/helper.py

```diff
@@ -110,8 +110,8 @@ def get_match_field_pb(self, table_name, match_field_name, value):
             lpm.mask = encode(value[1], bitwidth)
         elif match_type == p4info_pb2.MatchField.RANGE:
             lpm = p4runtime_match.range
-            lpm.low = encode(value[0], bitwidth)
-            lpm.high = encode(value[1], bitwidth)
+            lpm.low = (value[0]).to_bytes(bitwidth, byteorder='big')
+            lpm.high = (value[1]).to_bytes(bitwidth, byteorder='big')
```

Comment:
I am a bit confused by what this is supposed to fix. IIRC, the P4Runtime representation for range boundaries uses the same "encoding" as for other values. Which is why I would expect range values to be handled the same way as other values in this code.

Reply:
Thank you for pointing this out, you are right! 

I double-checked and it looks like the error I was getting earlier was due to missing `priority` field.